journalbeat/plan.sh

```diff
@@ -0,0 +1,35 @@
+pkg_name=journalbeat
+pkg_origin=core
+pkg_version="6.6.0"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_deps=(core/glibc core/systemd)
+pkg_build_deps=(core/go core/git core/make core/gcc core/systemd core/patchelf)
```

Comment:
systemd is included in `pkg_deps` so doesn't need to be included here.  pkg_deps are automatically included in the build process so you only need to declare them once. 

I don't see patchelf being used, I think it's safe to remove that dep.

Reply:
patchelf was brought in when I was struggling with systemd dep issues at runtime. Removed that and systemd and retested.